tests/foreman/api/test_convert2rhel.py

```diff
@@ -243,14 +252,15 @@ def test_convert2rhel_oracle(target_sat, oracle, activation_key_rhel, version):
             'inputs': {
                 'Activation Key': activation_key_rhel.id,
                 'Restart': 'yes',
+                'Data telemetry': 'yes',
```

Comment:
Q: What Data telemetry input does here?

Reply:
It is added as additional parameter in convert2Rhel job.